plugins/http_plugin/http_plugin.cpp

```diff
@@ -615,4 +631,13 @@ namespace eosio {
       return verbose_http_errors;
    }
 
+   http_plugin::get_supported_apis_result http_plugin::get_supported_apis()const {
+      get_supported_apis_result result;
```

Comment:
Is it recursive that the `get_supported_apis` endpoint will list itself?

Reply:
I also considered this, but it seems better to exclude `get_supported_apis` from the result on second thoughts.